Activity selection

NEEHITGOYAL opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description
Added solution of activity selection problem 
with the use of Dynamic Array of pairs in c++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#826

NEEHITGOYAL committed:
added activity selection code

NEEHITGOYAL committed:
add link to activity selection readme file

NEEHITGOYAL commented:
kindly review and merge this PR as part of hakin codes

NEEHITGOYAL committed:
Merge branch 'master' into ActivitySelection

NEEHITGOYAL commented:
KINDLY CHECK IT @HarshCasper

NEEHITGOYAL committed:
Merge branch 'master' into ActivitySelection

NEEHITGOYAL committed:
Update README.md

NEEHITGOYAL committed:
Merge branch 'master' into ActivitySelection

NEEHITGOYAL committed:
Merge branch 'master' into ActivitySelection

siddharth25pandey commented on C-Plus-Plus/cp/Activity_Selection.cpp:
```diff
@@ -0,0 +1,77 @@
+Explanation
+Person can only do 0th and 2nd activities.
+**/
+#include<iostream>
```
Add single header file
**bits/stdc++**

siddharth25pandey reviewed (changes requested):
Small modification is required
Else good to go!

NEEHITGOYAL committed:
Update Activity_Selection.cpp

added header file

Kajol-Kumari reviewed (changes requested):
@NEEHITGOYAL please make the following changes:

1. I am seeing many changes in the `readMe.md` file, I am not sure if all these are required for this PR, if not please revert them.

2. Your code indentation seems to be a bit off, can you please fit it?

NEEHITGOYAL committed:
Merge branch 'master' into ActivitySelection

NEEHITGOYAL committed:
Update Activity_Selection.cpp

done identation

NEEHITGOYAL commented:
@Kajol-Kumari due to so many days there were many pr merging and for resolving conflicts i have to make chages in readme and i still see there are two competitive coding links in readme i don't know who has done that i have made changes from my side and also idented the code you can re review

iamrajiv reviewed (changes requested):
@NEEHITGOYAL I can see so much edit in README file. Just add `- [ActivitySelection](cp/Activity_Selection.cpp)`.

NEEHITGOYAL committed:
fixing merge conflicts

Kajol-Kumari reviewed (changes requested):
@NEEHITGOYAL please use 
```
git fetch
git checkout upstream/master -- path/to/readme.md
```
this will revert all the changes and make the readme same as upstream/master(Here I  am assuming that you have added the original repo as upstream)

After doing this, just add your path file in readme, commit and push the changes.

For reference, do checkout [here](https://stackoverflow.com/questions/3334475/git-how-to-update-checkout-a-single-file-from-remote-origin-master)

NEEHITGOYAL commented:
Thankyou @Kajol-Kumari

NEEHITGOYAL committed:
updated readme

NEEHITGOYAL committed:
updated readme

NEEHITGOYAL committed:
resolved conflicts

NEEHITGOYAL commented:
i have updated readme same as it is now and added activity selection link and resolved all conflicts @Kajol-Kumari @iamrajiv  please check it and merge it thank you

Kajol-Kumari reviewed (approved):
LGTM :+1:

NEEHITGOYAL commented:
@siddharth25pandey @Abhijit2505  kindly merge it

siddharth25pandey reviewed (approved):
@NEEHITGOYAL Good to go!
Able to merge
Thanks!

siddharth25pandey commented:
@all-contributors please add @NEEHITGOYAL for his code.
